LayoutsCodelab/app/build.gradle

```diff
@@ -50,17 +50,17 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.6.0-alpha01'
-    implementation 'androidx.appcompat:appcompat:1.3.0-rc01'
+    implementation 'androidx.core:core-ktx:1.6.0-beta02'
+    implementation 'androidx.appcompat:appcompat:1.3.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation "androidx.compose.ui:ui:$compose_version"
-    implementation 'androidx.activity:activity-compose:1.3.0-alpha08'
+    implementation 'androidx.activity:activity-compose:1.3.0-beta01'
```

Comment:
The `constraintlayout-compose` dependency should've been updated but it's not. This might crash the app when used

Reply:
Doesn't seem to.